src/libertem/executor/dask.py

```diff
@@ -186,6 +186,33 @@ def get_available_workers(self):
             for worker in info['workers'].values()
         ])
 
+    def get_resource_details(self):
+        workers = self.get_available_workers()
+        details = {}
+
+        for worker in workers:
+            if worker.name.startswith("tcp"):
+                host_name = worker.host
+                resource = 'cpu'
```

Comment:
It's for the `dask-worker`, it doesn't support CUDA right?

Reply:
Well, since we support CUDA, in addition to just starting a plain `dask-worker`, we also have the `libertem-worker` command that allows to bring up worker processes with resource tags attached. See [the docker-compose config](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/docker/docker-compose.yml) for an example (although that disables CUDA explicitly)

Reply:
`dask-worker` is handled in the `else` part. I will add comments if this is confusing.

Reply:
Yeah - the critical part is that for a reader, it is not clear at all that a `name` starting with `tcp://...` means that we are dealing with `dask-worker`, I think that needs to be detailed in the comment.